### ISSUE TITLE
Development mode

### ISSUE THREAD

**Issue by joshuap** [honeybadger-io/honeybadger-js#929]

**Comment by subzero10** [comment]
Indeed, we can close this! For any future visitors, here's a [link](https://docs.honeybadger.io/lib/javascript/guides/environments-and-versions/#development-environments) to the documentation.
